units_of_measurement.go

```diff
@@ -0,0 +1,413 @@
+package sleet
```

Comment:
@nirajjayant we are starting to have a lot of files in this main sleet package, would it make sense to move some of them to `common`?

Reply:
Yeah I think currency, string stuff should move to common or utils or something